demo/config/application.rb

```diff
@@ -1,26 +1,28 @@
 require_relative "boot"
 
 require "rails/all"
-# require "sprockets/railtie"
 
 # Require the gems listed in Gemfile, including any gems
 # you've limited to :test, :development, or :production.
 Bundler.require(*Rails.groups)
-require "bootstrap_form"
+require "bootstrap_form" # FIXME: Why do we require this here?
```

Comment:
Have you tried without this line?

Reply:
Argh. I was sure I had removed the FIXME. Anyway, yes, I have another branch where I'm doing some small fix-ups, and it seems to run fine without this `require`. Maybe I'll merge that branch before I release.